### DISCUSSION TITLE
github action minutes cost?

### DISCUSSION THREAD

**Discussion by andy-brainome** [community/community#26980] [Actions]
<p>The blog post says that action minutes will be reduced.</p>
<p>How much are additional minutes?</p>

**Comment by dguo** [comment, marked as answer]
<p>You can find pricing details <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions" rel="nofollow noopener">here</a>. These are the current per minute prices for additional minutes, based on the OS:</p>
<table><tbody>
<tr>
<td>Linux</td>
<td>$0.008</td>
</tr>
<tr>
<td>macOS</td>
<td>$0.08</td>
</tr>
<tr>
<td>Windows</td>
<td>$0.016</td>
</tr>
</tbody></table>
